build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -72,7 +72,12 @@
       <MakeDir Directories="@(NuGetPackagesArchiveDirectory)"/>
 
       <DotNetNew ToolPath="$(Stage2Directory)"
-                 TemplateType="Web"
+                 TemplateType="--debug:reinit"
```

Comment:
that seems like a funny TemplateType. Please do change the implementation of DotNetNew if the current parameters no longer make sense...